src/Controls/src/Core/Device.cs

```diff
@@ -10,30 +10,41 @@
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../docs/Microsoft.Maui.Controls/Device.xml" path="Type[@FullName='Microsoft.Maui.Controls.Device']/Docs" />
+	//[Obsolete]
```

Comment:
Can't go 100% removed because of `Device.GetNamedSize` and potentially `Device.Styles`